docs/api.rst

```diff
@@ -8,6 +8,36 @@ Instrument
 .. autoclass:: pysat.Instrument
    :members:
 
+Instrument Methods
+------------------
+
+The following methods support the variety of actions needed
+by underlying pysat.Instrument modules.
+
+General
+^^^^^^^
+
+.. automodule:: pysat.instruments.methods.general
+   :members:
+
+.. _rst_general_data_cdaweb:
+
+netCDF Pandas
+^^^^^^^^^^^^^
+
+.. automodule:: pysat.instruments.templates.netcdf_pandas
+  :members: __doc__, init, load, list_files, download
+
+Instrument Templates
+--------------------
+
+General Instrument
+^^^^^^^^^^^^^^^^^^
+
+.. automodule:: pysat.instruments.templates.template_instrument
+   :members: __doc__, init, default, load, list_files, list_remote_files, download, clean
+
+
```

Comment:
NetCDF instrument should be here as well, or removed.

Reply:
I think these are both good